src/views/Print.js

```diff
@@ -45,12 +48,34 @@ class Print extends React.Component {
     constructor(props) {
         super(props)
         this.state = {
-            remainingQuota: 100,
+            remainingQuota: 0,
             fileNameDisplay: '',
-            files: []
+            files: [],
+            message: ""
         }
         this.upload = this.upload.bind(this);
         this.fileSelectHandler = this.fileSelectHandler.bind(this);
+        this.parseQuota = this.parseQuota.bind(this);
+    }
+
+    async componentDidMount() {
+        setTimeout(this.parseQuota, 2000);
+        let data = {method: "command",
+        command: 'pusage'}
+        ws.send(JSON.stringify(data))
+        var message = "";
+        ws.onmessage = msg => {
+            message = message + JSON.parse(msg.data).data
+            this.setState({message:message});
+        }
+        
+    }
+
+    parseQuota() {
+        var str = this.state.message.split('PS-printer paper usage:')[1];
```

Comment:
Use `JSON.parse` to decode stringified JSON objects

Reply:
const message = JSON.parse(this.state.message)
const paperUsage = message['PS-printer paper usage:']